### DISCUSSION TITLE
Slow loading problem when the page has multiple editors

### DISCUSSION THREAD

**Discussion by lppoi** [tinymce/tinymce#9126] [General]
Each Editor seems to reintroduce a copy of css

![image](https://github.com/tinymce/tinymce/assets/64509267/2e41947f-3bdc-4a53-be8b-6e69729f319b)

**Comment by lppoi** [comment]
How can I solve this problem?
local tinymce5.zip
@tinymce/tinymce-react: "^4.3.0"

**Comment by TheSpyder** [comment]
The content CSS is loaded in an iframe, so yes every iframe needs a new copy of the CSS (whether our default stylesheet or one included through the configuration).

Inline Editing mode doesn't use an iframe, so doesn't have this limitation, if you'd like to try it instead.
https://www.tiny.cloud/docs/tinymce/6/use-tinymce-inline/

**Reply by lppoi** [reply]
ok, thank you
